.github/workflows/release_please.yaml

```diff
@@ -0,0 +1,84 @@
+on:
+  push:
+    branches:
+      - main
+
+name: Run Release Please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # for googleapis/release-please-action to create release commit
+      pull-requests: write # for googleapis/release-please-action to create release PR
+
+    # Release-please creates a PR that tracks all changes
+    steps:
+      - uses: googleapis/release-please-action@a02a34c4d625f9be7cb89156071d8567266a2445 # v4
+        id: release
+        with:
+          token: ${{secrets.RELEASE_PLEASE_ACTION_TOKEN}}
+          target-branch: main
+
+      - name: Dump Release Please Output
+        env:
+          RELEASE_PLEASE_OUTPUT: ${{ toJson(steps.release.outputs) }}
+        run: |
+          echo "$RELEASE_PLEASE_OUTPUT"
+
+    outputs:
+      release_created: ${{ steps.release.outputs.releases_created }}
+      all: ${{ toJSON(steps.release.outputs) }}
+      paths_released: ${{ steps.release.outputs.paths_released }}
+
+  kotlin-release:
```

Comment:
If we're publishing artifacts individually, the release itself may need to be updated. [Here's how we did it in the Python contribs](https://github.com/open-feature/python-sdk-contrib/blob/main/.github/workflows/release.yml#L43). Perhaps we could do something similar here.